docs/docs/sbom/supported.md

```diff
@@ -0,0 +1,13 @@
+## Packages that support vulnerability scanning
+- [os packages][os_packages]
+- [language packages][os_packages]
```

Comment:
```suggestion
- [language packages][language_packages]
```

Reply:
I believe it is just a mistake. Fixed in https://github.com/aquasecurity/trivy/pull/2956/commits/c449e70edeab979239d538574dec9d6b3d0a929f

Reply:
I believe it is just a mistake. Fixed in https://github.com/aquasecurity/trivy/pull/2956/commits/c449e70edeab979239d538574dec9d6b3d0a929f

Reply:
yes, thanks for fixing this.